apps/web/components/common/FrontendExcelUploadDialog.tsx

```
@@ -0,0 +1,364 @@
+} from "@workspace/ui/components/table";
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  CardTitle,
+} from "@workspace/ui/components/card";
+import { FileUp, Loader2, CheckCircle } from "lucide-react";
+import * as XLSX from "xlsx";
+import { toast } from "sonner";
+
+interface TableHeader {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'TableHeader'. Consider to delete it or rename it.

'TableHeader' is defined here:


(lint/suspicious/noRedeclare)